src/test/resources/application-test.yml

```
@@ -4,4 +4,11 @@ spring:
     database-platform: org.hibernate.dialect.PostgreSQLDialect
     hibernate:
-      ddl-auto: create-drop
\ No newline at end of file
+      ddl-auto: create-drop
+  ai:
+    openai:
+      api-key: dummy
+    model:
+      embedding:
+        options:
+        model: text-embedding-3-small
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)